src/api/custom-schema/type-defs.ts

```diff
@@ -229,6 +234,40 @@ const typeDefs = `
     allCurrencies: [MappedCurrency]
   }
 
+  type Owner {
+    firstName: String
+    lastName: String
+    email: String
+  }
+
+  type ApplicationResponse {
+    id: String!
+    version: Int
+    createdAt: DateTime!
+    updatedAt: DateTime!
+    dealType: String!
+    submissionCount: Int
+    submissionDeadline: DateTime
+    submissionType: String
+    submissionDate: DateTime
+    status: String!
+    eligibility: Eligibility
+    nominatedLossPayee: NominatedLossPayee
+    policyContact: PolicyContact
+    owner: Owner
+    company: Company
+    business: Business
+    broker: Broker
+    buyer: Buyer
+    sectionReview: SectionReview
+    declaration: Declaration
+  }
```

Comment:
Half of these types don't seem to exist in `type-defs`, do we need them? Maybe could be simpler and only have high level/relevant fields?